iosApp/tools/sourcery/templates/AutoMockable.stencil

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import Shared
```

Comment:
For this import, it's better to use `autoMockableImports` in the file `.sourcery.yml`